src/indexer/parser.ts

```diff
@@ -268,21 +268,19 @@ export async function parseForIndexer(
             const { name } = attribute;
 
             if (name === 'title') {
-              state.meta.title ===
-                getStringValueFromAttribute({
-                  component: node,
-                  node: attribute,
-                  filename,
-                });
+              state.meta.title = getStringValueFromAttribute({
```

Comment:
This one was wow - how come we didn't notice? 😅

Reply:
😬